src/features/MovieList/MovieTile/styled.js

```diff
@@ -66,27 +68,31 @@ export const Year = styled.p`
     }
 `;
 
-export const TagsContainer = styled.div`
+export const TagsContainer = styled.ul`
     margin-top: 8px;
     display: flex;
     flex-wrap: wrap;
     gap: 8px;
+    padding: 0;
+    list-style: none;
+    margin-bottom: 0;
 `;
 
-export const Tag = styled.p`
+export const Tag = styled.li`
     margin: 0px;
     padding: 8px 16px;
     background-color: ${({ theme }) => theme.color.gray};
     border-radius: 5px;
     font-size: 14px;
+    
     @media (max-width: ${({ theme }) => theme.breakpoint.mobile}px) {
         padding: 4px 8px;
         font-size: 10px;
     }
 `;
 
 export const RatingContainer = styled.div`
-    margin-top: 39px;
+    margin-top: 14px;
```

Comment:
This value is different in Figma, is this bigger on purpose?

Reply:
i think 12 px like on figma is to close , but i can change to 12px maybe later we will changed it